src/docs/product/integrations/integration-platform/ui-components/alert-rule-action.mdx

```diff
@@ -0,0 +1,140 @@
+
+![Example of an alert rule UI component.](alert-rule-component-action-example.png)
+
+![Example of an alert rule settings modal.](alert-rule-component-modal-example.png)
+
+## Schema
+
+```javascript
+ {
+    "type": "alert-rule-action",
+    "title": <String>,
+    "settings": {
+        "type": "alert-rule-settings",
+        "uri": <URI>,
+        "required_fields": <Array<FormField>>,
+        "optional_fields": <Array<FormField>>,
+        "description": <String>
+    }
+}
+```
+
+## Attributes
+
+- `title` - (Required) The title shown in the UI component.
+- `uri` - (Required) Sentry will make a POST request to the URI when the User submits the form. If the services fails to process the request (status code >= 400), this component will bubble up the error to the User with the provided response text.
+- `required_fields` - (Required) List of [FormField](/product/integrations/integration-platform/ui-components/formfield) components the User is required to complete.
+- `optional_fields` - (Optional) List of [FormField](/product/integrations/integration-platform/ui-components/formfield) components the User may complete.
+- `description` - (Optional) Text that will be displayed above the form. Limited to 140 characters.
+
+## Example
+
+```javascript
+
+"elements": [
+    {
+        "type": "alert-rule-action",
+        "title": "Send Alerts to",
+        "settings": {
+        "type": "alert-rule-settings",
+        "uri": "/sentry/alert-rule",
+            "required_fields": [
+                {
+                    "type": "text",
+                    "label": "Channel",
+                    "name": "channel"
+                }
+            ]
+        }
+    }
+]
+```
+
+
+## Issue Alert Request Format
+
+When an issue alert fires, your service will need to read the settings from the alert payload. The `settings` are in `data.issue_alert.settings`. See the full [Issue Alert webhook documentation](/product/integrations/integration-platform/webhooks/#issue-alerts) for more information.
+
+### Payload
+
+```
+{
+    ...
+    "data": {
+        ...
+        "issue_alert": {
+            ...
+            "settings": [
+                {
+                    "name": "title",
+                    "value": "Ticket Title"
+                },
+                {
+                    "name": "description",
+                    "value": "Ticket Description"
+                }
+            ]
+        }
+    }
+}
+```
+
+## Metric Alert Request Format
+
+When a metric alert fires, your service will need to read the settings from the alert payload. The `settings` in this example are nested in `data.metric_alert.alert_rule.triggers[1].actions[0].settings`. There may be multiple `actions` and the index may not be 0 in your case. See the full [Metric Alert webhook documentation](/product/integrations/integration-platform/webhooks/#metric-alerts) for more information.
```

Comment:
> There may be multiple `actions` and the index may not be 0 in your case.

I think it might be more correct to say you'll have to identify which trigger and action contains your settings based on the webhook type, so `triggers[1]` and `actions[0]` may not be accurate in your case.